### PR TITLE
Add clockpicker definitions for v0.0.7

### PR DESCRIPTION
**Pull request by jfcere**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

### CODE REVIEW

File: types/clockpicker/tslint.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+    "dt-header": false,
```
Comment (plantain-00) [inline types/clockpicker/tslint.json]: `dt-header` rule shouldn't be disabled, 
usually there should be only `{"extends": "dtslint/dt.json"}` in this file

**Review by plantain-00 — CHANGES_REQUESTED** [review]

File: types/clockpicker/tslint.json (hunk 1 above)
Comment (jfcere) [reply types/clockpicker/tslint.json]: ClockPicker is only at version 0.0.7, therefore if I don't set this rule to false I would have to write 0.0 which I found weird in the index.d
ts file.

What are your thoughts on that matter?

Comment (plantain-00) [reply types/clockpicker/tslint.json]: I believe `0.0` is OK https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/isomorphic-fetch/index.d.ts#L1
`dt-header` rule is not just lint the version

Comment (jfcere) [reply types/clockpicker/tslint.json]: Alright I made the modification

**Review by plantain-00 — APPROVED** [review]

**Pull request merged**